website/docs/reference/resource-configs/trino-configs.md

```diff
@@ -36,67 +37,55 @@ For more information, refer to either the [Trino Connectors](https://trino.io/do
 }}
 ```
 
-## Materialization-specific configs
-
-### Seeds & Prepared Statements
-
-reference: [dbt docs: Seeds](https://docs.getdbt.com/docs/build/seeds)
-
-The `dbt seed` feature uses [Trino's prepared statements](https://trino.io/docs/current/sql/prepare.html).
+## Seeds and prepared statements
```

Comment:
thanks for this clean up, @nghi-ly! tbh, it's a lot of information, and it's actually relevant to more than just the starburst adapter (e.g. [SQL Server & Synapse](https://docs.getdbt.com/reference/resource-configs/mssql-configs#seeds)), so it can be re-homed in the future to make this page simpler.